### PR TITLE
feat(atom/radiobutton): add atom/radiobutton component

### PR DESCRIPTION
**Pull request by a-ferrante**

### CODE REVIEW

**Review by jcger — APPROVED** [review]

**Comment by juanmaguitar** [comment]
Is there a paper for this new component? What is then the role of the types `RADIO` & `CHECKBOX` in `AtomInput` → https://sui-components.now.sh/workbench/atom/input/demo ?

Captura de pantalla 2019-06-27 a las 8 25 29

**Comment by a-ferrante** [comment]
Same https://github.com/SUI-Components/sui-components/pull/722.

File: components/atom/radiobutton/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$maw-atom-radiobutton: 32px !default;
+
+.react-AtomRadiobutton {
```
Comment (rmoralp) [inline components/atom/radiobutton/src/index.scss:5]: Why `react` prefix instead `sui`?

File: components/atom/radiobutton/src/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+
+const AtomRadiobutton = ({
+  id,
+  value,
+  disabled,
+  checked,
+  onChange,
```
Comment (rmoralp) [inline components/atom/radiobutton/src/index.js:9]: I think that onChange should be required or just has an empty arrow function as a default value

**Review by rmoralp — CHANGES_REQUESTED** [review]
Why look&feel does not match with paper?

**Comment by JoanClaret** [comment]
as @juanmaguitar  is pointing out, since this component is already developed i don't get why to develop it again instead of use or improve the existing one.

**Comment by a-ferrante** [comment]
Based on the discussions held i will close this PR and we will revisit the inclusion of this components later once Arturo is back.

**Pull request closed without merging**